UACloudLibraryServer/GraphQL/UaCloudLibQuery.cs

```diff
@@ -29,37 +29,79 @@
 
-            Field<ListGraphType<ObjecttypeType>>(
-                "objecttype",
-                resolve: context => cloudLibRepo.GetObjectTypes()
-            );
+            Field<ListGraphType<ObjecttypeType>>("objecttype", resolve: context => cloudLibResolver.GetObjectTypes());
 
-            Field<ListGraphType<ReferencetypeType>>(
-                "referencetype",
-                resolve: context => cloudLibRepo.GetReferenceTypes()
+            Field<ListGraphType<ReferencetypeType>>("referencetype", resolve: context => cloudLibResolver.GetReferenceTypes());
+
+            Field<ListGraphType<VariabletypeType>>("variabletype", resolve: context => cloudLibResolver.GetVariableTypes());
+
+            Field<ListGraphType<NodesetType>>("nodeset", resolve: context => cloudLibResolver.GetNodesetTypes());
+            
+            Field<ListGraphType<CategoryType>>(
+                "category",
+                arguments: new QueryArguments(
+                    new QueryArgument<IntGraphType> { Name = "limit" },
+                    new QueryArgument<IntGraphType> { Name = "offset" },
+                    new QueryArgument<StringGraphType> { Name = "where" },
+                    new QueryArgument<StringGraphType> { Name = "orderby"}
+                ),
+                resolve: context =>
+                {
+                    int limit = context.GetArgument("limit", 1000000);
+                    int offset = context.GetArgument("offset", 0);
+                    string where = context.GetArgument("where", string.Empty);
+                    string orderby = context.GetArgument("oderby", string.Empty);
+                    return cloudLibResolver.GetCategoryTypes(limit, offset, where, orderby);
+                }
+            );
+                        
+            Field<ListGraphType<OrganisationType>>(
+                "organisation",
+                arguments: new QueryArguments(
+                    new QueryArgument<IntGraphType> { Name = "limit" },
+                    new QueryArgument<IntGraphType> { Name = "offset" },
+                    new QueryArgument<StringGraphType> { Name = "where" },
+                    new QueryArgument<StringGraphType> { Name = "orderby" }
+                ),
+                resolve: context =>
+                {
+                    int limit = context.GetArgument("limit", 1000000);
+                    int offset = context.GetArgument("offset", 0);
+                    string where = context.GetArgument("where", string.Empty);
+                    string orderby = context.GetArgument("oderby", string.Empty);
+                    return cloudLibResolver.GetOrganisationTypes(limit, offset, where, orderby);
+                }
             );
 
-            Field<ListGraphType<VariabletypeType>>(
-                "variabletype",
-                resolve: context => cloudLibRepo.GetVariableTypes()
+            Field<ListGraphType<AddressSpaceType>>(
+                "addressspace",

```

Comment:
Should this be addressSpace (camel cased)?

Reply:
Do you know what the GraphQL convention is? I thought everything should be lowercase?

Reply:
I think its a field name, so it should be camelCase:
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions

Reply:
orderBy should also be camel cased, and there are a few typos (oderby, missing "r").

Sorry these are trickling in: I just opened the schema in GraphIQl Explorer, instead of playground, and it makes the schema/query more approachable (to try it just add `app.UseGraphQLGraphiQL();` to startup.cs, and a GraphQL.Server.Ui.GraphiQL nuget ref).

Reply:
@barnstee agreed to fix before merge. Resolving.